pyatlan/generator/templates/referenceable_attributes.jinja2

```diff
@@ -72,8 +72,8 @@
     version: Optional[int] = Field(
         None, description='Version of this object.\n', example=2
     )
-    classifications: Optional[list[Classification]] = Field(
-        None, description="classifications"
+    atlan_tags: Optional[list[AtlanTag]] = Field(
+        None, description="classifications", alias="classifications"
```

Comment:
I would change the description to "Atlan tags" as well — we should only have `classification(s)` appear as an alias, and literally nowhere else (if possible)

Reply:
done